pkg/osutil/user.go

```diff
@@ -149,7 +150,19 @@ func LimaUser(limaVersion string, warn bool) *user.User {
 			if err != nil {
 				logrus.Debug(err)
 			} else {
-				home += ".linux"
+				// Trim mount prefix within Subsystem
+				// cygwin/msys2 cygpath could have prefix for drive mounts configured via /etc/fstab
+				// wsl wslpath could have prefix for drive mounts configured via [automount] section in wsl.conf
+				drivePath, err := ioutilx.WindowsSubsystemPath(filepath.VolumeName(limaUser.HomeDir) + "/")
+				if err != nil {
+					logrus.Debug(err)
+				} else {
+					prefix := path.Dir(strings.TrimSuffix(drivePath, "/"))
+					if prefix != "/" {
+						home = strings.TrimPrefix(home, prefix)
+					}
+					home += ".linux"
+				}
```

Comment:
This complicated logic should have unit tests

Reply:
I would like to write a test for this one, but this have some heavy dependencies. It needs cygpath, which is mostly fine for Windows specific cases, but for wslpath - one needs WSL distribution as it is available only inside container. 

At some point I had idea to remove this part entirely and rely only on hand coded cygpath alternative in the code below this (the second conditional branch), as for the purpose of creating home directory it was good enough.

I will create a ticket, where I summarize what options are available to improve situation.